src/app/(core)/v/[slug]/player.tsx

```
@@ -11,6 +11,129 @@
 
+function parseOffsetValue(value: string): number | undefined {
+  // if there are no colons, assume its seconds
+  if (/^\d+$/.test(value)) return parseInt(value, 10);
+
+  // Supports HH:MM:SS, MM:SS, SS
+  // If it's not in the format, return undefined
+  if (!/^([0-5]?[0-9]:){0,2}[0-5][0-9]$/.test(value)) return undefined;
+
+  return value
+    .split(":")
+    .reduce((acc, cur) => (acc = acc * 60 + parseInt(cur, 10)), 0);
```

Biome

[error] The assignment should not be in an expression.

The use of assignments in expressions is confusing.
Expressions are often considered as side-effect free.

(lint/suspicious/noAssignInExpressions)